src/equations/compressible_navier_stokes.jl

```diff
@@ -26,6 +26,25 @@ struct NoSlip{F}
     boundary_value_function::F # value of the velocity vector on the boundary
 end
 
+"""
+    struct SlipWall
```

Comment:
Does it make sense for naming consistency to call this just 'Slip', or to add 'NoSlipWall' as an alias for 'NoSlip'?